pkg/backend/httpstate/client/client.go

```diff
@@ -1127,6 +1127,30 @@ func (pc *Client) UpdateStackTags(
 	return pc.restCall(ctx, "PATCH", getStackPath(stack, "tags"), nil, tags, nil)
 }
 
+func (pc *Client) UpdateStackDeployment(ctx context.Context, stack StackIdentifier,
+	deployment apitype.DeploymentSettings,
+) error {
+	// TODO(german): validate deployment
```

Comment:
Is this worth doing now?  Otherwise can we write down which validations we expect to do here?  I guess the backend will also do its own validation anyway, so we would be getting an error here if the deployment isn't valid?

Reply:
We should probably remove the TODO. No need for us to validate CLI side of the service can do it better.